docs/docs/studio.md

```diff
@@ -89,37 +201,82 @@ token.
-  few more free ports available next to it.
-
 ### Database File 
 Studio stores the ingested telemetry to `studio/prisma.db` within the
 `.redwood` folder. You should not need to touch this file other than if you
 wish to delete it to erase any existing telemetry data.
 
-## Availability
-Along the release of Redwood v7, Studio has been rewritten and is available as
-a stable version. Just run `yarn rw studio` to start it! Prior to RW v7 Studio
-was available as an experimental feature. If you're still not using the stable
-version of RW v7, see this forum topic for information on the old experimental
-version of Studio
-https://community.redwoodjs.com/t/redwood-studio-experimental/4771
+## Troubleshooting
+
+We hope you don't encounter any issues, but here are some ways to troubleshoot Studio. As always, you can find us on Discourse Forums or GitHub issues to report problems.
+
+### Concurrency Limit Reached
+
+If you see an 'OpenTelemetry concurrency limit reached' error in your logs, consider increasing the `concurrencyLimit` value in your OpenTelemetry configuration.
+
+```tsx
+
+// api/src/opentelemetry.ts
+
+// ...
+
+const studioPort = getConfig().studio.basePort
+
+const exporter = new OTLPTraceExporter({
+  // Update this URL to point to where your OTLP compatible collector is listening
+  // The redwood development studio (`yarn rw studio`) can collect your
+  // telemetry at `http://127.0.0.1:<PORT>/.redwood/functions/otel-trace`
+  // (default PORT is 4318)
+  url: `http://127.0.0.1:${studioPort}/.redwood/functions/otel-trace`,
+  concurrencyLimit: 64, // <- increase 
+})
+
+// ...
+```
+
+### Timeouts
+
+If your application experiences any timeouts while sending telemetry data to Studio, you can increase the timeout by adding a `timeoutMillis` value (the default is 10 seconds).
+
+```tsx
+
+// api/src/opentelemetry.ts
+
+// ...
+
+const studioPort = getConfig().studio.basePort
+
+const exporter = new OTLPTraceExporter({
+  // Update this URL to point to where your OTLP compatible collector is listening
+  // The redwood development studio (`yarn rw studio`) can collect your
+  // telemetry at `http://127.0.0.1:<PORT>/.redwood/functions/otel-trace`
+  // (default PORT is 4318)
+  url: `http://127.0.0.1:${studioPort}/.redwood/functions/otel-trace`,
+  concurrencyLimit: 64,
+  timeoutMillis: 30000, // <-- set a timout greater than 10 seconds
+})
+
+// ...
+```
+ 
+### Connection Refused
+
+If your application cannot connect to Studio, please ensure the following:
+
+- Studio should be running before you launch your app's development server.
+
+- If you stop Studio while your app is running in development mode, remember to restart your app after restarting Studio.
+
+
+## Legacy Studio
+
+Studio has been rewritten and is available as part of RedwoodJS v7. Previously it was available as an [experimental feature](https://community.redwoodjs.com/t/redwood-studio-experimental/4771). 
```

Comment:
I think we also include a section explaining that once Studio has been activated on your project, you always need to run it, otherwise, you'll get an error when you run `yarn rw dev` because it's looking for Studio.